Update influxdb.markdown with measurement_attr option

markbergsma opened the pull request:
Documentation for the new measurement_attr option.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
PR home-assistant/core#36020 introduces a new config option to select InfluxDB measurement names based on State object attributes.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: home-assistant/core#36020
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

markbergsma committed:
Update influxdb.markdown with measurement_attr option

Documentation for the new measurement_attr option introduced in home-assistant/core#36020

klaasnicolaas commented on source/_integrations/influxdb.markdown:
```diff
@@ -77,14 +77,19 @@ max_retries:
   default: 0
+measurement_attr:
+  type: string
+  description: State object attribute to use as measurement name. Possible values: `unit_of_measurement` (default), `domain`, `entity_id`
```
```suggestion
  description: "State object attribute to use as measurement name. Possible values: `unit_of_measurement`, `domain` or `entity_id`."
```

markbergsma committed:
Update source/_integrations/influxdb.markdown

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

markbergsma committed:
Update measurement_attr possible values

As discussed in home-assistant/core#36020 we implement domain__device_class
instead of just domain.
